### PR TITLE
Add com.github.bytezz.IPLookup

### PR DESCRIPTION
**Pull request by Bytezz**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Bytezz** [comment]
bot, build com.github.bytezz.IPLookup

File: com.github.bytezz.IPLookup.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "com.github.bytezz.IPLookup",
```
Comment (hfiguiere) [inline com.github.bytezz.IPLookup.json:2]: it should be `io.github`

File: com.github.bytezz.IPLookup.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                    "type" : "git",
+                    "url" : "https://github.com/Bytezz/IPLookup-gtk.git",
+                    "tag" : "v0.1.1"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
+    }
```
Comment (hfiguiere) [inline com.github.bytezz.IPLookup.json:41]:
```suggestion
    ]
```

File: com.github.bytezz.IPLookup.json (hunk 1 above)
Comment (Bytezz) [reply com.github.bytezz.IPLookup.json:2]: Changing that I would have to change a lot of things. Opened a [PR](https://github.com/flathub/flatpak-builder-lint/pull/139) for exception as suggested [here](https://github.com/flathub/flatpak-builder-lint/issues/15#issuecomment-1278743255).

Next project I start will be "io.github".

**Comment by Bytezz** [comment]
Since no response for a while, I converted id to io.github.

**Comment by hfiguiere** [comment]
so you expected an unpaid volunteer to answer your question between 0:40 and 9:15 ? I'd really hate to have you as a paying client.

**Comment by Bytezz** [comment]
> so you expected an unpaid volunteer to answer your question between 0:40 and 9:15 ? I'd really hate to have you as a paying client.

I'm sorry, I don't know where are you from and your current timezone.
I didn't mean to be rude, so I apologize if I was.

Anyway, I took the opportunity to fix the manifest (code and file names) as it should be. I had inadvertently skipped the part about the com.github id inside the docs.

**Pull request closed without merging**